src/escape.rs

```diff
@@ -104,7 +108,27 @@ static HTML_ESCAPES: [&'static str; 6] = [
         "&gt;"
     ];
 
-pub fn escape_html<W>(mut w: W, s: &str, secure: bool) -> io::Result<()>
+#[cfg(all(target_arch = "x86_64", feature="simd"))]
+pub fn escape_html<W: Write>(mut w: W, s: &str) -> io::Result<()> {
+    let bytes = s.as_bytes();
+    let mut mark = 0;
+
+    scan_simd(bytes, 0, |i| {
+        let replacement = HTML_ESCAPES[HTML_ESCAPE_TABLE[bytes[i] as usize] as usize];
+        w.write_all(&bytes[mark..i])?;
+        w.write_all(replacement.as_bytes())?;
+        mark = i + 1;  // all escaped characters are ASCII
+        Ok(())
+    })?;
+    w.write_all(&bytes[mark..])
+}
+
+#[cfg(not(all(target_arch = "x86_64", feature="simd")))]
+pub fn escape_html<W: Write>(w: W, s: &str) -> io::Result<()> {
+    escape_html_secure(w, s, false)
+}
+
+pub fn escape_html_secure<W>(mut w: W, s: &str, secure: bool) -> io::Result<()>
```

Comment:
Not clear yet why we have this "secure" flag.

Reply:
Me neither, it's never used. Hysterical raisins, I guess. Can remove.